planner/core/plan_to_pb.go

```diff
@@ -212,6 +212,22 @@ func checkCoverIndex(idx *model.IndexInfo, ranges []*ranger.Range) bool {
 		if len(rg.LowVal) != len(idx.Columns) {
 			return false
 		}
+		if !rg.LowExclude {
```

Comment:
Do we need to check whether null exists if `rg.LowExclude or rg.HighExclude`?

Reply:
Updated.